credentials/src/test/kotlin/web5/sdk/credentials/PresentationExchangeTest.kt

```diff
@@ -247,7 +257,7 @@ class PresentationExchangeTest {
 
       val vcJwt = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
 
-      assertThrows<SignatureException> {
+      assertThrows<IllegalStateException> {
```

Comment:
now failing with IllegalStateException because the `vcJwt` in this test does not contain header.kid